python/tests/test_storage_connector.py

```diff
@@ -192,6 +193,21 @@ def test_from_response_json_basic_info(self, backend_fixtures):
 
 
 class TestSnowflakeConnector:
+    def test_read_query_option(self, mocker):
+        mocker.patch("hsfs.engine.get_instance", return_value=spark.Engine())
+        mock_engine_read = mocker.patch("hsfs.engine.spark.Engine.read")
+
+        snowflake_connector = storage_connector.SnowflakeConnector(
+            id=1, name="test_connector", featurestore_id=1, table="snowflake_table"
+        )
+        query = "select * from table"
+        snowflake_connector.read(query=query)
+
+        print(mock_engine_read.call_args)
```

Comment:
Just some nit stuff - do we need this print here?

Reply:
oh no, forgot to remove it after debugging.